src/components/NameField.js

```diff
@@ -8,14 +8,14 @@ export const NameField = ({ name, label, form, classes, autoComplete }) => {
   const layout = useLayout();
   const [, setData] = useData();
 
-  const { control, errors } = form;
+  const { control, formState: { errors } } = form; // in react-hook-form 7, errors are stored in formState
```

Comment:
Where is this one used? it feels redondant with the "normal" TextField.js, is this still used?

Reply:
Here https://github.com/fixthestatusquo/proca/blob/65638673d92d10ebf8b1574c265c9a3aff821c52/src/components/donate/Stripe.js#L154, donate/Stripe.js/line 155. Do you want me to replace it with TextField?

Reply:
yeah, and remove that NameField, one less file !